hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/CacheHitMissNonStrictTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.hibernate;
+
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.annotation.QuickTest;
+import org.hibernate.cache.access.AccessType;
+import org.hibernate.cfg.Environment;
+import org.hibernate.stat.CollectionStatistics;
+import org.hibernate.stat.SecondLevelCacheStatistics;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(QuickTest.class)
+public class CacheHitMissNonStrictTest extends HibernateStatisticsTestSupport {
```

Comment:
This class needs some documentation. I have no clue what you are testing here and what the purpose of strict is.